osbs/tekton.py

```diff
@@ -460,6 +463,8 @@ def update_labels(self, labels):
         exc_msg = f"Can't update labels on pipeline run '{self.pipeline_run_name}', " \
                   f"because it doesn't exist"
         response_json = self._check_response(response, msg)
+        logger.debug("labels after update: %s", self.labels)
```

Comment:
Checking the labels before and after is 2 extra HTTP queries on every `update_labels()` call.

Reply:
yeah but it will be removed, once we figure out root cause for labels not being updated correctly